### ISSUE TITLE
Toggle input screenreader issue

### ISSUE THREAD

**Issue by LindaMeijering** [formkit/formkit#1381]
### Reproduction
https://formkit.link/2dd72ae7aa7f3a49f6317c3283384c43

### Describe the bug

Screen reader reads "checkbox not checked" without the label for the toggle. The element has two label elements, maybe that's the cause?

### Environment

• OS: [e.g. iOS]
• Browser [e.g. chrome, safari]
• Version [e.g. 22]

**Comment by sashamilenkovic** [comment]
@LindaMeijering I'm having a hard time reproducing this, or perhaps I'm not quite understanding the issue. Whether I use a label or not, Apple's VoiceOver appears to be correctly reading whether the toggle is checked or not. I was hoping you could provide additional details and/or further explain the issue.

**Comment by LindaMeijering** [comment]
I have added two checkboxes so you can see how it is to navigate between them with just using tab on the keyboard. I use the screenreader extension in chrome. When selecting a checkbox and moving from one to the other with just tab and shift+tab it only reads "checkbox checked" or "checkbox not checked" but it does not read the label of the box. in this case I would expect the screenreader to also read "Formkit Input".

**Comment by sashamilenkovic** [comment]
@LindaMeijering Got it, thank you. Could you also share which screenreader extension you are using (just to be sure we're on the same page).

**Comment by LindaMeijering** [comment]
@sashamilenkovic 
![image](https://github.com/formkit/formkit/assets/116559427/3a59bcf5-7745-4a47-a23c-2b7c7c07a848)
https://chromewebstore.google.com/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en-US&utm_source=ext_sidebar

**Comment by sashamilenkovic** [comment]
@LindaMeijering I can replicate this issue using your screenreader, and yes, your assumption that the double label might be causing an issue is correct (if I remove it it works fine). However, I think we want the source of truth to be native screen readers as this works fine on Apple Voiceover. We'll close this issue until there is a reproduction of this not working on perhaps Window's Narrator.
